utility/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
In general, we should avoid defaulting visibility to public. We should keep the visibility of library as restrictive as they need to be (I think keeping these module libraries publicly visible is fine, but that should be defined per-library rather than for the whole file).

Reply:
Done!